src/main/java/uk/gov/justice/hmpps/prison/api/resource/OffenderResource.java

```diff
@@ -241,6 +242,20 @@ public InmateDetail transferOutPrisoner(
         @RequestBody @NotNull @Valid final RequestToTransferOut requestToTransferOut) {
         return prisonerReleaseAndTransferService.transferOutPrisoner(offenderNo, requestToTransferOut);
     }
+    @ApiResponses({
```

Comment:
API required just for testing, but can the basis of a future API to transfer to court but will required more analysis and testing